dace/codegen/targets/rtl.py

```diff
@@ -244,10 +269,33 @@ def generate_rtl_inputs_outputs(self, sdfg, tasklet):
             # add element index
             idx_str += "[{}:0]".format(int(total_size / vec_len) * 8 - 1)
             # generate padded string and add to list
-            outputs.append(", output reg{padding}{idx_str} {name}".format(
-                padding=" " * (12 - len(idx_str)), idx_str=idx_str, name=inp))
+            outputs.append(", output reg{padding} m_axis_{name}_tvalid".format(
+                padding=" " * 12, name=inp))
+            outputs.append(
+                ", output reg{padding}{idx_str} m_axis_{name}_tdata".format(
+                    padding=" " * (12 - len(idx_str)),
+                    idx_str=idx_str,
+                    name=inp))
+            outputs.append(", input {padding} m_axis_{name}_tready".format(
+                padding=" " * 16, name=inp))
+            outputs.append(
+                ", output reg{padding}[3:0] m_axis_{name}_tkeep".format(
+                    padding=7 * " ", name=inp))
+            outputs.append(", output reg{padding} m_axis_{name}_tlast".format(
+                padding=" " * 12, name=inp))
```

Comment:
...what is going on here? 😆

Reply:
Pretty "printing" of the ports:
```
, output reg             m_axis_b_tvalid
, output reg      [31:0] m_axis_b_tdata
, input                  m_axis_b_tready
, output reg       [3:0] m_axis_b_tkeep
, output reg             m_axis_b_tlast
);
```
The same happens for the inputs. Could just be emitted without the spacing, making the Codegen more readable? Or it could be more structured?

Reply:
Ok, maybe factor this into a separate function that states clearly what it does, then call it for each line?

Reply:
Fixed in https://github.com/carljohnsen/dace/commit/c45585997749ebec935cebbb300fbd12b0134b55